clang/test/CodeGen/WebAssembly/wasm-externref-novec.c

```diff
@@ -0,0 +1,18 @@
+// RUN: %clang_cc1 -O2 -triple wasm32-unknown-unknown-wasm -target-feature +reference-types -emit-llvm -o - %s | FileCheck %s
```

Comment:
```suggestion
// RUN: %clang_cc1 -O2 -triple wasm32-unknown-unknown -target-feature +reference-types -emit-llvm -o - %s | FileCheck %s
```

Reply:
By the way, this command succeeds even without this patch. The test should be something that fails now and succeeds with the fix.